src/canvas-view/canvas-editor/canvas-editor.less

```diff
@@ -39,7 +39,7 @@
       vertical-align: middle;
     }
     .show-hide-connectors-label {
-      vertical-align: text-bottom;
+      vertical-align: 2px;
```

Comment:
Removing the vertical-align setting all together seems to make it line up better.